app/views/partials/foyer/capture-montant-ressource.html

```diff
@@ -8,14 +8,14 @@ <h3 class="panel-title">
   <div class="panel-body">
     <div class="row">
       <div class="col-sm-3" ng-repeat="month in months">
-        <label for="{{ index + ressource.type.id + month.id }}">
+        <label for="{{ ressource.type.id }}_month{{ $index + 1 }}_{{ index }}">
           {{ month.label|uppercaseFirst }}
         </label>
         <span class="input-group">
           <input
             type="number"
             class="form-control text-right"
-            id="{{ index + ressource.type.id + month.id }}"
+            id="{{ ressource.type.id }}_month{{ $index + 1 }}_{{ index }}"
```

Comment:
Huh? Weird. That shouldn't be a problem:

> the id attribute value must be unique
> There are no other restrictions on what form an ID can take; in particular, IDs can consist of just digits, **start with a digit**, start with an underscore, consist of just punctuation, etc.

[Spec](https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute)